recipes/mmtf-pyspark/meta.yaml

```diff
@@ -0,0 +1,36 @@
+package:
+  name: mmtfpyspark
+  version: "0.2.0"
+
+source:
+  git_url: https://github.com/sbl-sdsc/mmtf-pyspark
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy x.x
```

Comment:
The `numpy x.x` syntax has been deprecated and the current recommendation is:

https://conda-forge.org/docs/meta.html#building-against-numpy